web/src/activity_ui.ts

```diff
@@ -59,6 +59,18 @@ export function clear_for_testing(): void {
     user_filter = undefined;
 }
 
+function update_status_circles(): void {
+    $(".user_circle").each(function () {
+        const user_id = Number.parseInt($(this).attr("data-status-sync-user-id") ?? "", 10);
```

Comment:
Let's use "presence" rather than "status" here; this is about presence/availability, not the "available" status.

`data-presence-indicator-user-id` and `update_presence_indicators()` might be appropriate.

Reply:
Done